### PR TITLE
feat: PerOpInflator

### LINKED ISSUE #2: Implement `submitBatch` in the BundleBulker
Currently you can only submit 1 userop in the BundleBulker

It would be useful to be able to submit multiple userops, each with its own specified inflator (which can differ for ops in the same batch)

Maybe could also have another one for the case when all the ops in the batch have the same inflator, so you don't need to specify the inflator for each and you can save some calldata

### PR DESCRIPTION
**Pull request by dcposch**
This is a second bundle `IInflator`. The first one was a proof-of-concept for Daimo transfers.

**This one takes a list of userops, each compressed with their own individual `IOpInflator`, concatenated.** It doesn't apply any additional cross-op compression--it simply expands `list[(op inflator id, compressed op)]` to `list[op]`.

Fixes #2

### CODE REVIEW

**Comment by dcposch** [comment]
## Down to 119 byte transfers

See https://github.com/daimo-eth/bulk/pull/3/files#diff-7b93e8b1d879dc608a53c881f55c24448923314c28c8af20d78dca2453be58fcR41-R55

- Eliminate WebAuthn challenge entirely. Reconstruct it via the (computed) userOpHash.
- Replace from, to with `DaimoNameRegistry` lookups. In future, this could be further optimized to a 4-byte index, but given that Daimo names are on average 5-6 bytes the savings are small.
- Remove sponsoring paymaster signature. We need to use the `tx.origin` trick instead--otherwise, sponsoring paymaster ends up nearly doubling the size of the transfer. ERC20 paymaster unaffected.

**Comment by dcposch** [comment]
pairs with https://github.com/daimo-eth/daimo/pull/583

File: src/DaimoOpInflator.sol [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "account-abstraction/interfaces/IEntryPoint.sol";
+import "p256-verifier/utils/Base64URL.sol";
+
+interface INameReg {
+    function resolveAddr(bytes32 name) external view returns (address);
+}
+
+/// Inflates an op containing a Daimo ERC20 transfer.
+/// This reduces calldata usage from ~1.5kb to ~400 bytes.
```
Comment (nalinbhardwaj) [inline src/DaimoOpInflator.sol]: should this be lower now?

**Review by nalinbhardwaj — APPROVED** [review]
lgtm, looks like CI might need some updates, stubbed #5

**Pull request merged**